src/oemof/solph/_models.py

```diff
@@ -352,14 +372,35 @@ def __init__(self, energysystem, discount_rate=None, **kwargs):
                 f"you have to specify the `discount_rate` attribute."
             )
             warnings.warn(msg, debugging.SuspiciousUsageWarning)
+        elif (
+            isinstance(energysystem, list)
+            and energysystem[0].periods is not None
+        ):
+            self.discount_rate = 0.02
+            msg = (
+                f"By default, a discount_rate of {self.discount_rate} "
+                f"is used for a multi-period model. "
+                f"If you want to use another value, "
+                f"you have to specify the `discount_rate` attribute."
+            )
+            warnings.warn(msg, debugging.SuspiciousUsageWarning)
```

Comment:
If I'm not mistaken, this is the same as above. Couldn't we add a tiny method for this that we call twice to avoid code duplication?